build/targets/KoreBuild.DefaultBuildSettings.props

```diff
@@ -1,4 +1,16 @@
 <Project>
+  <!-- pickup build number from CI if available -->
+  <PropertyGroup Condition=" '$(BuildNumber)' == '' ">
+    <!--
+      TeamCity=https://confluence.jetbrains.com/display/TCD10/Predefined+Build+Parameters
+      AppVeyor=https://www.appveyor.com/docs/environment-variables/
+      Travis=https://docs.travis-ci.com/user/environment-variables/
+    -->
+    <BuildNumber Condition=" '$(BUILD_NUMBER)' != '' ">$(BUILD_NUMBER)</BuildNumber>
+    <BuildNumber Condition=" '$(APPVEYOR_BUILD_NUMBER)' != '' ">$(APPVEYOR_BUILD_NUMBER)</BuildNumber>
```

Comment:
(shrug) It's a "nice to have" that may be useful one day. But if you really want, I'll remove it. It doesn't make a difference at the moment.